e2e/package-lock.json

```diff
@@ -53,7 +53,6 @@
       "resolved": "https://registry.npmjs.org/@babel/generator/-/generator-7.5.0.tgz",
       "integrity": "sha512-1TTVrt7J9rcG5PMjvO7VEG3FrEoEJNHxumRq66GemPmzboLWtIjjcJgk8rokuAS7IiRSpgVSu5Vb9lc99iJkOA==",
       "dev": true,
-      "optional": true,
```

Comment:
This is a problem; these `optional` properties were added recently with a package update PR (https://github.com/chef/automate/pull/2611).
You need to upgrade your version of node to current (12.13.1) then re-run the `npm install` to fix this.